### ISSUE TITLE
Add 1.12 data to minecraft-data : Bot crashes when other entities hold terracotta blocks

### ISSUE THREAD

**Issue by VTCAKAVSMoACE** [PrismarineJS/mineflayer#533]
Minecraft 1.12 added tons of blocks, including the terracotta blocks. It seems that the client doesn't like it so much.

```
assert.js:81
  throw new assert.AssertionError({
  ^
AssertionError: item with id 235 not found
    at new Item (/home/minecraft/[redacted]/node_modules/prismarine-item/index.js:22:10)
    at Function.Item.fromNotch (/home/minecraft/[redacted]/node_modules/prismarine-item/index.js:61:10)
    at Client.<anonymous> (/home/minecraft/[redacted]/node_modules/mineflayer/lib/plugins/entities.js:77:57)
    at emitTwo (events.js:106:13)
    at Client.emit (events.js:191:7)
    at Parser.deserializer.on (/home/minecraft/[redacted]/node_modules/minecraft-protocol/src/client.js:102:12)
    at emitOne (events.js:96:13)
    at Parser.emit (events.js:188:7)
    at readableAddChunk (/home/minecraft/[redacted]/node_modules/readable-stream/lib/_stream_readable.js:211:18)
    at Parser.Readable.push (/home/minecraft/[redacted]/node_modules/readable-stream/lib/_stream_readable.js:170:10)
    at Parser.Transform.push (/home/minecraft/[redacted]/node_modules/readable-stream/lib/_stream_transform.js:123:32)
```

**Comment by VTCAKAVSMoACE** [comment]
This does not happen when using `createClient` with minecraft-protocol.

**Comment by rom1504** [comment]
huh lol, how could I forget this before merging https://github.com/PrismarineJS/mineflayer/pull/524

I'll extract the 1.12 data

**Comment by rom1504** [comment]
fixed, you can use release 2.4.1
